### DISCUSSION TITLE
docs/develop/getting-started/addon-basics/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#198] [Page comments]
# docs/develop/getting-started/addon-basics/

https://scratchaddons.com/docs/develop/getting-started/addon-basics/

**Comment by tondawill** [comment]
gjju

**Comment by RussianRScratcher** [comment]
how do i can create a addon

**Reply by BroJac5246** [reply]
@RussianRScratcher see https://scratchaddons.com/docs/develop/getting-started/creating-an-addon/

**Comment by asdfperson135** [comment]
thank you!

**Reply by faretek1** [reply]
idk why but i cant upvote :Sob:
